OSDOCS-10545 add important admonition under topic title with link to … (#88568)
openshift/openshift-docs: networking/ovn_kubernetes_network_provider/configuring-egress-ips-ovn.adoc

As a cluster administrator, you can configure the OVN-Kubernetes Container Network Interface (CNI) network plugin to assign one or more egress IP addresses to a namespace, or to specific pods in a namespace.

[IMPORTANT]
====

> In an installer-provisioned infrastructure cluster, do not assign egress IP addresses to the infrastructure node that already hosts the ingress VIP. For more information, see the Red{nbsp}Hat Knowledgebase solution link:https://access.redhat.com/solutions/7069883[POD from the egress IP enabled namespace cannot access OCP route in an IPI cluster when the egress IP is assigned to the infra node that already hosts the ingress VIP]

Author: @lahinson  added the period at end of sentence. Once all tests pass, we should be good, hopefully

> In an installer-provisioned infrastructure cluster, it is recommended to not assign egress IP addresses to the infra node that already hosts the ingress VIP. For more information, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/7069883[POD from the egress IP enabled namespace cannot access OCP route in an IPI cluster when the egress IP is assigned to the infra node that already hosts the ingress VIP]

Reviewer: nit; infrastructure

> In an installer-provisioned infrastructure cluster, it is recommended to not assign egress IP addresses to the infrastructure node that already hosts the ingress VIP. For more information, see the Red Hat Knowledgebase solution link:https://access.redhat.com/solutions/7069883[POD from the egress IP enabled namespace cannot access OCP route in an IPI cluster when the egress IP is assigned to the infra node that already hosts the ingress VIP]

Reviewer 2: ```suggestion
In an installer-provisioned infrastructure cluster, do not assign egress IP addresses to the infrastructure node that already hosts the ingress VIP. For more information, see the Red{nbsp}Hat Knowledgebase solution link:https://access.redhat.com/solutions/7069883[POD from the egress IP enabled namespace cannot access OCP route in an IPI cluster when the egress IP is assigned to the infra node that already hosts the ingress VIP].
```
- Keep the terms `Red` and `Hat` together: https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces
- Unless for some reason it's important to use the phrase `it is recommended to not`, the simpler, `do not` should work.

Author: @lahinson  working on the changes now

Author: @lahinson Changes have been made and ready for you to review
